test/integration_test_helper.rb

```diff
@@ -10,28 +10,41 @@
 require 'database_cleaner'
 require 'active_support_test_case_helper'
 require 'minitest/retry'
+require 'selenium/webdriver'
+
 retry_count = (ENV['MINITEST_RETRY_COUNT'] || 3).to_i rescue 1
 Minitest::Retry.use!(retry_count: retry_count) if retry_count > 1
 
 Minitest::Retry.on_consistent_failure do |klass, test_name|
   Rails.logger.error("DO NOT IGNORE - Consistent failure - #{klass} #{test_name}")
 end
 
-Capybara.register_driver :poltergeist do |app|
-  opts = {
-    # To enable debugging uncomment `:inspector => true` and
-    # add `page.driver.debug` in code to open webkit inspector
-    # :inspector => true
-    :js_errors => true,
-    :timeout => 60,
-    :extensions => ["#{Rails.root}/test/integration/support/poltergeist_onload_extensions.js"],
-    :phantomjs => File.join(Rails.root, 'node_modules', '.bin', 'phantomjs')
-  }
-  Capybara::Poltergeist::Driver.new(app, opts)
+if ENV["JS_TEST_DRIVER"] == 'selenium_chrome'
+  Selenium::WebDriver::Chrome.driver_path = ENV['TESTDRIVER_PATH'] || File.join(Rails.root, 'node_modules', '.bin', 'chromedriver')
+  Capybara.register_driver :selenium_chrome do |app|
+    options = Selenium::WebDriver::Chrome::Options.new
+    options.args << '--disable-gpu'
+    options.args << '--no-sandbox'
+    options.args << '--window-size=1024,768'
+    Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+  end
+else
+  Capybara.register_driver :poltergeist do |app|
+    opts = {
+      # To enable debugging uncomment `:inspector => true` and
+      # add `page.driver.debug` in code to open webkit inspector
+      # :inspector => true
+      :js_errors => true,
+      :timeout => 20,
+      :extensions => ["#{Rails.root}/test/integration/support/poltergeist_onload_extensions.js"],
+      :phantomjs => File.join(Rails.root, 'node_modules', '.bin', 'phantomjs')
+    }
+    Capybara::Poltergeist::Driver.new(app, opts)
+  end
 end
 
-Capybara.default_max_wait_time = 30
-Capybara.javascript_driver = :poltergeist
+Capybara.default_max_wait_time = 20
+Capybara.javascript_driver = ENV['JS_TEST_DRIVER'].try(:to_sym) || :poltergeist
```

Comment:
you can probably move this inside the if else, so you don't have to check the same condition twice?